src/imitation/algorithms/adversarial.py

```diff
@@ -379,7 +360,7 @@ def _build_disc_feed_dict(
         n_gen = len(gen_samples.obs)
 
         if expert_samples is None:
-            expert_samples = self._expert_dataset.sample(n_gen)
+            expert_samples = self.expert_dataset.sample(n_gen)
```

Comment:
Why did I remove the underscore here? Should check whether `gen_algo`, `discrim`, and `expert_dataset` are/should be used outside of this class. If not, add back the underscore.

Reply:
Note to self: gen_algo and discrim are (or probably will be) accessed outside the trainer (e.g. discrim is accessed to save the algorithm). It makes sense for them to be stable/publicly accessible. expert_dataset is not used elsewhere and should have an underscore.